src/objects/Points.js

```diff
@@ -55,7 +55,7 @@ Points.prototype = Object.assign( Object.create( Object3D.prototype ), {
 			inverseMatrix.getInverse( matrixWorld );
 			ray.copy( raycaster.ray ).applyMatrix4( inverseMatrix );
 
-			var localThreshold = threshold / ( ( this.scale.x + this.scale.y + this.scale.z ) / 3 );
+			var localThreshold = 3 * threshold / ( this.scale.x + this.scale.y + this.scale.z );
```

Comment:
I think this should be
```javascript
var localThreshold = threshold / ( 0.5 * ( this.scale.x + this.scale.y )  );
```
And even that is a hack.

Reply:
`Points` are 2D only.

Reply:
Sorry. I was confusing something...

Reply:
... but my personal preference is to leave the code in this file as-is for clarity, and change the other file to match.

Reply:
Ok, but I guess this should be mentioned as a caveat in the docs, right?

Reply:
I don't follow you... just leave the code like so:
```javascript
var localThreshold = threshold / ( ( this.scale.x + this.scale.y + this.scale.z ) / 3 );
```

Reply:
I mean we should warn the user that this might lead to unexpected results. If the `Line` or `Points` have a parent that has scale (0.1, 1, 1) for example then the hit accuracy will be quite off.

Reply:
Non-uniform scale of parental objects has always been a problem with three.js as it leads to shearing. Here, the issue is non-uniform scale of the object itself -- which is generally not a problem, but it is a problem in this code as written.

If you want to propose a performant improvement, go for it! :)